e2e/testdata/fn-render/fn-failure/.expected/config.yaml

```diff
@@ -16,4 +16,4 @@
 # non-zero exit code and no changes in the resources.
 exitCode: 1
 stdErr: "httpbin-gen:4:1: got newline, want primary expression"
-stdOut: "running function \"gcr.io/kpt-fn/starlark:unstable\": FAILED"
+stdOut: "[RUNNING] \"gcr.io/kpt-fn/starlark:unstable\"\n[FAIL] \"gcr.io/kpt-fn/starlark:unstable\""
```

Comment:
you can do multi-line string in YAML to make this more readable?

Reply:
done